apps/passport-client/src/loadInitialState.ts

```diff
@@ -24,6 +30,24 @@ export async function loadInitialState(): Promise<AppState> {
   const self = loadSelf();
 
   const pcds = await loadPCDs(self);
+
+  if (
+    pcds &&
+    self &&
+    (!self.semaphore_v4_commitment || !self.semaphore_v4_pubkey)
+  ) {
+    const semaphoreV4IdentityPCD = pcds.getPCDsByType(
```

Comment:
This janky logic based on the first element in an array should probably be in only one place, in a helper function.  findIdentityPCD() in user.ts does that for V3.